packages/amplify/amplify_flutter/lib/src/dart_impl.dart

```diff
@@ -30,6 +30,7 @@ class AmplifyDartImpl extends AmplifyClass {
     );
     await Future.wait(
       [
+        ...API.plugins,
```

Comment:
I think you will have to have similar line in hybrid_impl.dart (this bug is from code I wrote).

Reply:
You may need to rebase on `next` since I think I deleted this file